tests/unit/library.cairo

```diff
@@ -77,6 +78,8 @@ namespace it {
 
         %{
             stop_prank = start_prank(ids.caller) 
+            mock_call(ids.to, "onERC3525Received", [ids.IERC3525_RECEIVER_ID])
+            mock_call(ids.to, "supportsInterface", [1])
```

Comment:
to be consistent with the other test, do we want to change `[1]` to `[ids.TRUE]`  ?

Reply:
good catch